include/llbuild/Basic/CrossPlatformCompatibility.h

```diff
@@ -0,0 +1,28 @@
+//===- PlatformUtility.h ----------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines cross platform wrappers around non-cross platform types.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CrossPlatformCompatibility_h
+#define CrossPlatformCompatibility_h
+
+#if defined(_WIN32)
+#include <cstdint>
+typedef int32_t llbuild_pid_t;
```

Comment:
Actually we might want to use `ifdef _MSC_VER` here, since this typedef is only needed for MSVC. MinGW provides sys/types.h which already defines pid_t to an appropriate value.